litewallet/Platform/Extensions.swift

```diff
@@ -558,3 +558,17 @@ extension Dictionary where Key: ExpressibleByStringLiteral, Value: Any {
 		return jstring
 	}
 }
+
+extension UInt64 {
```

Comment:
It's recommended to move the UInt64 extension to a dedicated file named something like `UInt64+SafeArithmetic.swift`.